digitalpublicgoods/digit-public-finance-management.json

```diff
@@ -0,0 +1,96 @@
+{
+  "name": "DIGIT Public Finance Management",
+  "clearOwnership": {
+    "isOwnershipExplicit": "Yes",
+    "copyrightURL": "https://ifix.digit.org/"
+  },
+  "platformIndependence": {
+    "mandatoryDepsCreateMoreRestrictions": "No",
+    "isSoftwarePltIndependent": "",
+    "pltIndependenceDesc": ""
+  },
+  "documentation": {
+    "isDocumentationAvailable": "Yes",
+    "documentationURL": [
+      "https://ifix.digit.org/platform/strategy-and-approach/functional-specifications",
+      "https://ifix.digit.org/platform/strategy-and-approach/technical-specification"
+    ]
+  },
+  "NonPII": {
+    "collectsNonPII": "Yes",
+    "checkNonPIIAccessMechanism": "Yes",
+    "nonPIIAccessMechanism": "Dashboards and data dump downloads from dashboards in XLS format."
+  },
+  "privacy": {
+    "isPrivacyCompliant": "Yes",
+    "privacyComplianceList": [
+      "Information Technology Act, 2000 (as interpreted by the Supreme Court of India in the Puttaswamy - 1 judgment)"
+    ],
+    "adherenceSteps": [
+      "Steps taken for adherence are in compliance with the Information Technology Act, 2000 (as interpreted by the Supreme Court of India in the Puttaswamy - 1 judgment) Steps are -: Purposeful collection, Legitimate interest in data collection &  Data minimization."
+    ]
+  },
+  "standards": {
+    "supportStandards": "Yes",
+    "standardsList": [
+      "Open API Specifications",
+      "JSON",
+      "REST"
+    ],
+    "evidenceStandardSupport": [
+      "https://redocly.github.io/redoc/?url=https://raw.githubusercontent.com/egovernments/iFix-Dev/develop/domain-services/ifix-master-data-service/ifix-master-data-service-1.0.0.yaml",
+      "https://github.com/egovernments/iFix-Dev/blob/master/domain-services/fiscal-event-service/fiscal-event-service-1.0.0.yaml",
+      "https://github.com/egovernments/iFix-Dev/blob/master/domain-services/fiscal-event-post-processor/fiscal-event-post-processor-1.0.0.yaml",
+      "https://github.com/egovernments/iFix-Dev/blob/master/domain-services/ifix-department-entity-service/ifix-department-entity-service-1.0.0.yaml"
+    ],
+    "implementBestPractices": "Yes",
+    "bestPracticesList": [
+      "Interoperable, Standards based, Minimalism, Open Source, Unbundled, Secure & Reliable, Scalable, Ease of use, Inclusive"
+    ]
+  },
+  "doNoHarm": {
+    "preventHarm": {
+      "stepsToPreventHarm": "Yes",
+      "additionalInfoMechanismProcessesPolicies": "DIGIT-iFix is built with data privacy and security features. We follow best practices during development and deployment to ensure security. It does no harm. "
```

Comment:
Please verify which best practices during development & deployment help to ensure security and in the long run do no harm. Thanks.

Reply:
The following mechanisms have been provided to ensure security:

- **Authentication** - Can be done using username password or one time passwords that are sent to the mobile numbers of users.
- **Password and Session Expiration Policy** -periodic password updation and session logins to ensure user authentication. Password expiration is work in progress.
- **Encryption** - Sensitive Data can be encrypted before being stored.
- **Role Based Authorisation** - Roles can be configured which limits the features and functionality available to the user.
- **Data Isolation** - Users are bound to the tenant and the user is isolated to access data
available only for that tenant. Tenants can be a department or geography etc.
Tenants can be hierarchical.
● **Non Repudiation** - To ensure non-repudiation, a signed audit log is maintained for all
the changes.